examples/mlj/binary.jl

```diff
@@ -0,0 +1,29 @@
+module Binary
+
+using CatBoost.MLJCatBoostInterface
+using DataFrames
+using MLJBase
+using PythonCall
+
+# Initialize data
+cat_features = [0, 1]
+train_data = DataFrame([["a", "a", "c"], ["b", "b", "d"], [1, 4, 30], [4, 5, 40],
+                        [5, 6, 50], [6, 7, 60]], :auto)
```

Comment:
This is generating a warning because `scitype(["a", "a", "c"])` is `Textual` but, according to your implementation's `input_scitype`, only `Continuous`, `OrderedFactor`, or `Count` columns are allowed. 

Now you have specified `cat_features = [0, 1]` (more on this hyper-parameter in my comments below) from which I infer you mean `["a", "a", "c"]` to be categorical, presumably `Multiclass` and not `OrderedFactor`. So you probably want to replace with `coerce(["a", "a", "c"], Multiclass)` in this test (`coerce` is exported by `ScientificTypes` and also `MLJBase`).

Reply:
Thank you for explaining this. I understand now. This should be resolved in my latest commit.